### ISSUE TITLE
Question about setting poll_interval in the sidekiq.yml

### ISSUE THREAD

**Issue by abepark01** [gzigzigzeo/sidekiq-grouping#35]
Hello!

I set `:poll_interval` in the `config/sidekiq.yml` but doesn't seem to be taking effect when I restart the worker process.
I am using `sidekiq` 5.1.3 and `sidekiq-grouping` 1.0.9

**config/sidekiq.yml**
```
grouping:
  :poll_interval: 60

production:
  :concurrency: 2

staging:
  :concurrency: 2
```

**Log**
```
[Sidekiq::Grouping] Started polling batches every 3 seconds
```
So, it looks like the default of 3 seconds is being used instead.

Any suggestions here?
Also, I was a bit confused about what `poll_interval` does vs `batch_flush_interval`; how are they different?

Thank you!
Abraham

**Comment by sergeyk** [comment]
I have the same question. Has anyone else encountered this?

**Comment by sergeyk** [comment]
@abepark01 actually you should make sure that you're running the latest (1.0.9) version -- upgrading fixed for me

**Comment by abepark01** [comment]
@sergeyk yes, I reported that I was using 1.0.9 as well

**Comment by le0pard** [comment]
Should be fixed in 1.0.10 by this commit - 208412a8ecc50300c5adae70e2679096884334b8
